packages/bitcoin/src/bitcoin.utils.ts

```diff
@@ -36,11 +36,11 @@ export function initBitcoinAccount(derivationPath: string, policy: string): Bitc
  * has a number of networks, its often only necessary to consider the higher
  * level concepts of mainnet and testnet
  */
-export const bitcoinNetworkToCoreNetworkMap: Record<BitcoinNetworkModes, NetworkModes> = {
+export const bitcoinNetworkToCoreNetworkMap: Record<BitcoinNetworkModes, BitcoinNetworkModes> = {
```

Comment:
@kyranjamie , I'm having a go at solving https://github.com/leather-io/extension/issues/4874 but I'm not sure if this is correct. 

Is `bitcoinNetworkToCoreNetworkMap` and `bitcoinNetworkModeToCoreNetworkMode` intended to:

1. block us from using `regtest` + `signet` in some cases
2. not setup correctly with all possible networks

In the extension I can see cases where we call `bitcoinNetworkModeToCoreNetworkMode` and in the bug report you say to write a test to make sure the BTC send flow should work when choosing the `regtest` network. 

Maybe my change here is incorrect but instead I need to change the `extension` to stop using `bitcoinNetworkModeToCoreNetworkMode` for:
- `useBtcSendForm`
- `useSendInscriptionForm`

Reply:
@fbwoolf , I should have tagged you yesterday. Maybe you can help me understand the history here and how I should fix this?

Reply:
The problem of this issue doesn't lie in this map/function, which serves the purpose of turning a specific network into the less granular, broader concept of mainnet/testnet.

What is likely happening is that we're writing logic considering only mainnet/testnet, when we need logic specific to regtest. If that's the case, the logic needs to change to never call this function in the first place, not change this fn.

Reply:
OK thanks 👍